package.json

```diff
@@ -22,6 +26,12 @@
     "npm",
     "javascript"
   ],
+  "//dependencies": {
+    "//": "Comments about why some dependencies are at specific versions that may issue warnings.",
+    "glob": "Intentionally pinned to v7 to support node >= 0.8.",
+    "inflight": "Dependency of glob v7.",
```

Comment:
i don't see us depending on inflight at all?

Reply:
It is a dependency of the preferred version of glob and showed up as a warning during npm install, so I thought it made sense to mention it here.

Reply:
Warnings from transitive deps aren't actionable for anyone and should always be ignored, no need to mention it i think.